tasks/generateFeatureIndex.js

```diff
@@ -0,0 +1,74 @@
+require('colors');
+const glob = require('glob');
+const fs = require('fs');
+const path = require('path');
+
+const FEATURES_PROJECT_DIR = path.join('sauce', 'features');
+const FEATURES_INDEX_PROJECT_PATH = path.join(FEATURES_PROJECT_DIR, 'index.js');
+
+function run(callback) {
+  glob(`${FEATURES_PROJECT_DIR}/*/**/index.js`, (error, files) => {
+    if (error) return callback(error);
+
+    let imports = [];
+    let featureNames = [];
+
+    files.forEach((filePath) => {
+      let filePathSplit = filePath.split(path.sep);
+      let projectFeaturePath = filePathSplit.slice(0, filePathSplit.length - 1).join(path.sep);
+
+      // up one directory to use the project path
+      let featureSetting = require(`../${projectFeaturePath}/settings.js`); // eslint-disable-line global-require
+
+      // features/index will source from the features folder, so remove
+      // `sauce/features` from the path here.
+      let featureIndexPath = filePathSplit.slice(2, filePathSplit.length - 1).join(path.sep);
+
+      let importLine = `import ${featureSetting.name} from './${featureIndexPath}';\n`;
```

Comment:
hmm, this is going to be an import line in the actual JS file. Will that not work on windows?